css/properties.json

```diff
@@ -6381,7 +6381,7 @@
     "groups": [
       "CSS Masking"
     ],
-    "initial": "center",
+    "initial": "0% 0%",
```

Comment:
this is already landing in https://github.com/mdn/data/commit/2987ea9827a8e8a29cbae0ed0a6bf02dffab3301